website/styles.css

```
@@ -10,12 +10,30 @@ body {
     fixed 10px 10px / 20px 20px radial-gradient(#d1d1d1 1px, transparent 0);
 }
 
+/* Overriding the default colour schemas for light mode*/
+.light [data-language], .light .nextra-code-block>pre {
+  background-color: #0a4b6c  !important;
+}
+.light .nextra-code-block>div {
+  background-color: #0a4b6c   !important;
+  color: aliceblue !important;
+}
+.light button[title="Copy code"], button[title="Toggle word wrap"] {
+    color: aliceblue !important;
```

```suggestion
  color: aliceblue !important;
```

Some extra spaces here